plugins/modules/pacemaker_cluster.py

```diff
@@ -210,6 +214,8 @@ def main():
         set_cluster(module, 'offline', timeout, force)
         cluster_state = get_cluster_status(module)
         if cluster_state == 'offline':
+            if module.check_mode:
+                module.exit_json(changed=True)
```

Comment:
```suggestion
        if module.check_mode:
                module.exit_json(changed=True)
        set_cluster(module, 'offline', timeout, force)
        cluster_state = get_cluster_status(module)
        if cluster_state == 'offline':
```

Shouldn't your code look like this? Otherwise, you would take the cluster offline even in check mode. And won't take it online again.

Or do I get this wrong?

Reply:
Yes, that's right. I thought I did it that way, but must have gotten confused...

(I reduced the indent in your suggestion so I can apply it without further post-processing :) )